Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
 WORKDIR /go/src/github.com/openshift/cluster-samples-operator
 COPY . .
-RUN go build ./cmd/cluster-samples-operator
+RUN ./build-go.sh
```

Comment:
i'm pretty sure, though not positive, that @smarterclayton doesn't want us customizing these dockerfiles

(though i do not know what that means for how we're supposed to inject git commit version info into our binaries)

Reply:
(maybe customizing the first stage is ok and just not the second stage)

Reply:
It’s ok for customizing this line.  This line is yours